### PR TITLE
Use Wayland by default

### PR DESCRIPTION
**Pull request by travier**

### CODE REVIEW

**Comment by tsdgeos** [comment]
What about the reasons given in https://github.com/flathub/org.kde.okular/pull/34 ?

Has that been fixed¿

**Comment by travier** [comment]
> What about the reasons given in #34 ?
> 
> Has that been fixed¿

Good catch. I don't think so. Will remove.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Okular (org.kde.okular)
+
+## Permissions rationale
+
+This application requests the following additional permissions:
+
+- Access to the host filesystem: `--filesystem=host`.
+  This is required for drag & drop support and to enable openinf files from
```
Comment (Erick555) [inline README.md:8]: typo `openinf` -> `opening`. 

Also `--filesystem=host` doesn't open access to `/tmp`. IIRC recent firefox doesn't use `/tmp` anymore. I would just end this descripton on _This is required for drag & drop support_.

**Comment by Pointedstick** [comment]
I tested this on both X11 and Wayland in Fedora 35 and it works fine.

File: README.md (hunk 1 above)
Comment (travier) [reply README.md]: Thanks!

**Pull request merged**